src/Compiler/CompilerUtils.hpp

```diff
@@ -42,9 +42,10 @@ void setTargetArch(const std::string &arch);
 void setTargetTriple(const std::string &triple);
 void setOptLevel(const onnx_mlir::OptLevel level);
 // Set compile context according to the (key, value) fields passed.
-void setCompileContext(mlir::MLIRContext &context,
-    const llvm::SmallVector<std::pair<onnx_mlir::OptionKind, std::string>, 4>
-        options);
+using CompilerOptionList =
+    llvm::SmallVector<std::pair<onnx_mlir::OptionKind, std::string>, 4>;
+void setCompileContext(
```

Comment:
nit: I would put this using declaration at the top of the header to give it more "visibility".

Reply:
make sense